wandb/sdk/artifacts/artifact.py

```diff
@@ -1122,13 +1123,14 @@ def new_file(
             )
             raise e
 
-        self.add_file(path, name=name)
+        self.add_file(path, name=name, policy="immutable")
 
     def add_file(
         self,
         local_path: str,
         name: Optional[str] = None,
         is_tmp: Optional[bool] = False,
+        policy: Optional[str] = "mutable",
```

Comment:
could you make this one a Literal, also it doesn't seem to be optional.

Reply:
Or did you mean that a Literal[] cannot be optional?